content/devops/foundational-short-course/5-devops-teams/_index.md

```diff
@@ -0,0 +1,22 @@
+---
+content_type: topic
+ready: true
+title: DevOps Teams
+---
+
+DevOps is a cultural mindset, but we see a lot of “DevOps” jobs out there. Here’s a couple of them and what they mean:
+
+## IT Operations
+Also known as sysadmin, the responsibilities include deploying and managing the IT infrastructure – network, servers, storage, backup, monitoring system’s behavior and performance, contracts with vendors, IT budgeting etc.
+
+## DevOps Engineer
+Basically the same responsibilities as IT operations, but with a mindset of enabling the development teams to be as autonomous as possible, giving them a platform to build, test, deploy and monitor their own applications. 
+
+## DevSecOps
+A DevOps person with one foot in security, working with standardizing environments, access and privileges, network access, system’s isolation, cryptography, data encryption, systems integrity, bug bounty, security tests during continuous integration tests etc.
+
+## DataOps:
+A DevOps person with data management knowledge, enabling collaboration between operations and data scientist/analyst.
+
+## Site Reliability Engineer (SRE):
+A team of engineers focused on automation, system design and system resilience, responsible for the system’s availability, performance, efficiency, observability, emergency response and capacity planning. It has also been considered “DevOps”, but the SRE team is more focused into building reliable systems, where the DevOps focus is more broadly.
```

Comment:
*DevOps focus is more broad.

Reply:
I didn't get this one 🤔

Reply:
IMHO DevOps is meant to include all teams responsible for software development, but as with all buzzwords, its definition varies. 

I'll leave this content like this. When talking about job description, "DevOps Engineer" _usually_ means "IT Ops but hey I know Jenkins".

Reply:
Thanks for the meme, I loved it 😂